presto-docs/src/main/sphinx/release/release-0.254.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.254
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where queries that have both remote functions and a local function with only constant arguments could throw an IndexOutOfBoundException during planning. The bug was introduced in release 0.253 by :pr:`16039`.
+* Add documentation for Glue Catalog support in Hive. :doc:`/connector/hive`.
+* Add iceberg connector.
+* Add support to fragment result caching for unnest.
+* Add :func:`poisson_cdf` and :func:`inverse_poisson_cdf` functions.
+* Add memory tracking in ``TableFinishOperator`` which can be enabled by setting the ``table-finish-operator-memory-tracking-enabled`` configuration property to ``true``.
```

Comment:
What are the user-facing implications.  why would a user turn this on?

Reply:
If there is GC issue on coordinator, this will help to debug whether stats collection uses a lot memory.